### ISSUE TITLE
InfiniteBoost: building infinite ensembles with gradient descent

### ISSUE THREAD

**Issue by marugari** [dmlc/xgboost#2384]
InfiniteBoost: building infinite ensembles with gradient descent
https://arxiv.org/abs/1706.01109
Code (not found)
https://github.com/arogozhnikov/infiniteboost

It looks good.
